### DISCUSSION TITLE
Move Weblate GitHub fork to be based of other repo

### DISCUSSION THREAD

**Discussion by ann0see** [WeblateOrg/weblate#8112] [Q&A]
Currently, the Weblate repository of our project (jamulussoftware/jamulus) is based on a contributors repository (ignotus666/jamulus), not the official one. Is it possible to access the weblate repository and delete + recreate it based on jamulussoftware/jamulus?

Also, is it possible to rewrite history on this repository? If I as maintainer try to force push to weblate/jamulus while a PR is open, I get a permission error.

Repo for reference:
https://github.com/weblate/jamulus/

**Comment by nijel** [comment]
Change the Weblate configuration first to point to the new repo, once that is done, please write us at care@weblate.org to delete and recreate the repo. 

Weblate does force push for its changes automatically, so you just need to get rewritten history to Weblate.

**Comment by ignotus666** [comment]
The weird thing here is that the component was never configured to point to my repo. What's more, it doesn't pull updates from my repo but from the one it's configured to use (jamulussoftware/jamulus). It _appears_ to be based on my repo ([see here](https://github.com/weblate/jamulus/tree/weblate-jamulus-jamulus-app)), but behaves correctly (i.e., pulls and pushes changes to jamulussoftware/jamulus).

**Reply by ann0see** [reply]
@ignotus666 I think we're good to send the E-Mail now.

**Comment by ann0see** [comment]
This is now resolved.
